dotcom-rendering/src/components/CartoonComponent.tsx

```diff
@@ -0,0 +1,96 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, Pillar } from '@guardian/libs';
+import { Hide } from '@guardian/source-react-components';
+import type { CartoonBlockElement } from '../types/content';
+import { Caption } from './Caption';
+import { Picture } from './Picture';
+
+type Props = {
+	element: CartoonBlockElement;
+};
+
+export const CartoonComponent = ({ element }: Props) => {
+	const smallVariant = element.variants.find(
+		(variant) => variant.viewportSize === 'small',
+	);
+	const largeVariant = element.variants.find(
+		(variant) => variant.viewportSize === 'large',
+	);
+
+	const getFullCredit = (credit: string, source: string) => {
+		if (credit && source) {
+			return `${credit} / ${source}`;
+		} else if (!credit && source) {
+			return source;
+		} else if (credit && !source) {
+			return credit;
+		} else return '';
+	};
+
+	const fullCredit = getFullCredit(element.credit, element.source);
+
+	return (
+		<>
+			<div
+				css={css`
+					position: relative;
+
+					img {
+						height: 100%;
+						width: 100%;
+					}
+				`}
+			>
+				<Hide until="desktop">
+					{largeVariant?.images.map((image) => {
+						return (
+							<Picture
+								master={image.url}
+								role={element.role}
+								format={{
+									display: ArticleDisplay.Standard,
+									design: ArticleDesign.Standard,
+									theme: Pillar.News,
+								}}
+								alt={element.alt}
```

Comment:
Hm, I don't think we would expect the large variant to have more than one image, but not sure if there's any limitation on that so that a large variant can strictly only have one - do you know, @rebecca-thompson?

Reply:
Just from an over the desk chat - large variants with multiple images is an edge case that we do have to handle, so yep, will adjust this!